docs/src/reference/riklet.md

```diff
@@ -0,0 +1,41 @@
+# Riklet
+
+[Rust in Kube (RIK)](https://github.com/dev-sys-do/rik) node agent
+
+## Faas Configuration
+
+**Prerequisite**: You need firecracker in yout PATH.
+
+Here is a list of **required** environment variables to run riklet:
+
+| Environment Variable   | Description                                                                                                             | Default |
+| ---------------------- | ----------------------------------------------------------------------------------------------------------------------- | ------- |
+| `IFACE`                | Network interface connected to the internet                                                                             | ""      |
+| `IFACE_IP`             | IP of the Network interface connected to the internet                                                                   | ""      |
+| `FIRECRACKER_LOCATION` | Path to the firecracker binary                                                                                          | ""      |
+| `KERNEL_LOCATION`      | Path to the kernel location                                                                                             | ""      |
+| `SCRIPT_LOCATION`      | Path to the [script](https://github.com/polyxia-org/rik/blob/main/scripts/setup-host-tap.sh) that create tap interface. | ""      |
+
+To run riklet with FAAS configuration.
+
+```bash
+sudo riklet --kernel-path ${KERNEL_LOCATION} \
+            --ifnet ${IFACE} \
+            --ifnet-ip ${IFACE_IP} \
+            --script-path ${SCRIPT_LOCATION}
```

Comment:
Hummm i don't know maby @kalil-pelissier have an idea

Otherwise, if I keep this script line, I leave all env variables linked to this [comment](https://github.com/rik-org/rik/pull/123#discussion_r1191344569) else i juste talk about FIRECRACKER_LOCATION env.

what do you think @AlexandreBrg ?

Reply:
Just checked, `script-path` has been removed when we went to iptables home-made crate.

Leave all environment variables, just tell that they are alternative ways to configure riklet.

Reply:
ahah okok mb I misread your comment :melon: